lib/user.rb

```diff
@@ -1,4 +1,14 @@
 class User < ActiveRecord::Base
   validates_presence_of :email, :password
   validates_uniqueness_of :email
+
+  def user_songs
+    user_song_arr = []
+    Song.all.each do |song|
+      if song.suggested_by.id == self.id
```

Comment:
Can we replace this with `.suggested_by_id` instead of `.suggested_by.id` and save query?